### ISSUE TITLE
Does the number of morphology files in a given path influence whether a program runs properly or runs at all?

### ISSUE THREAD

**Issue by riddick-the-furyan** [BlueBrain/NeuroTS#67]
I tried synthesizing some pyramidal neurons last night, but I ran into a TypeError because a 1D vector was expected. I know it is not the source because I already synthesized some neocortex interneurons with the code, so I copied and pasted it into a seperate .ipynb file and changed the directories.

I had 40 morphology files for the interneurons and 670 for the pyramidal. I tried mixing around the neurite_types to see if it would work depending on which were present. Still no luck. So I gave up on it and decided I'll start again tomorrow.

This morning I decide to remove 665 files and only leaving 5 to see if it will run. I used neurite_types = ["basal_dendrite", "apical_dendrite"] in the distribution function. I did the same thing last night and it didn't work. So, I ran the program, and it worked after I removed all of those files.

Here is the source code:

```
import json
from pathlib import Path

import neurots
from neurots import extract_input


def run(distr_dir, params_dir, neuron_dir, data_dir):
    for i in range(0,50,1):
        """Run the example for extracting inputs for synthesis."""
        # Generate distribution from directory of neurons

        distr_name = "test_distr_{}.json".format(i)
        params_name = "test_params_{}.json".format(i)
        neuron_name = "test_neuron_{}.swc".format(i)

        distr = extract_input.distributions(
            data_dir / "neurons",
            neurite_types = ["basal_dendrite", "apical_dendrite"],
            feature="path_distances", 
            diameter_model="default",
        )

if __name__ == "__main__":
    data_dir = Path("C:\\Users\\user\\Documents\\Experiment_01\\Data\\Human\\Neocortex_pyramidals")
    params_dir = Path("C:\\Users\\user\\Documents\\Experiment_01\\Parameters\\Neocortex_pyramidal")
    distr_dir = Path("C:\\Users\\user\\Documents\\Experiment_01\\Distributions\\Neocortex_pyramidal")
    neuron_dir = Path("C:\\Users\\user\\Documents\\Experiment_01\\Synthesized\\Neurons\\Neocortex_pyramidal")

    run(distr_dir, params_dir, neuron_dir, data_dir)
    print("DONE!!!")
```

It could also be due to the files themselves and how much or what data is present in them but I don't have information on that, but I got both morphologies from neuromorpho.org.

**Comment by riddick-the-furyan** [comment]
I just checked the path to the files and it is definently the files that are the issue.

**Comment by riddick-the-furyan** [comment]
I just checked the path to the files and it is definently the files that are the issue.

**Comment by riddick-the-furyan** [comment]
I just checked the path to the files and it is definently the files that are the issue.

**Comment by adrien-berchet** [comment]
Hi @riddick-the-furyan 
If you have some time could you send me the error and the files that generated it so I can see what's happening? (for the files you can just send me the procedure to download them from neuromorpho.org)
Thanks

**Comment by riddick-the-furyan** [comment]
Hello @adrien-berchet 
It was a type error because a 1D vector was expected. For the interneurons and pyramidals I used ones from the human neocortex. To download them hover over the browse tab, click by animal species, on the pie chart click the human slice, then press CTRL + F and type interneuron and pyramidals because there are a lot of filles. 

After that download all of the human neocortex pyramidal and interneuron morphologies, separately. When I deleted pyramidal cells I saved the first five on my directory with default viewing on windows and deleted the rest. Apologies for the late reply.
